packages/phone-number-privacy/signer/README.md

```diff
@@ -124,6 +131,18 @@ Then check on the service to make sure its running:
 
 `docker logs -f {CONTAINER_ID_HERE}`
 
+#### Key rotations
+
+After a key resharing, signers should rotate their key shares as follows:
+
+1. Store the new key share in the keystore according to the naming convention specified in the [Keystores](#keystores) section above.
+2. Increment `PHONE_NUMBER_PRIVACY_LATEST_KEY_VERSION` or `DOMAINS_LATEST_KEY_VERSION` as appropriate. This will instruct the signer to prefetch this new key version the next time it starts up, but there is no need to restart the signer at this point.
```

Comment:
I worry about unintended reboots or scaling up which would trigger restart of the service. WDYT about indexing the keys on polynomial and the Combiner can request a specific key from the signer. This forces the Combiner deployer to not forget the polynomial and the Combiner can easily roll back to an old version upon failure without having to coordinate with the signers.

Reply:
Just documenting that we discussed this in a meeting and determined that given the length of the public polynomials it would be difficult to index this way. It's worth noting also that the combiner will be able to roll back to an older version so long as the signers have not removed that key version from their vaults